x/evm/types/params.go

```diff
@@ -176,8 +176,8 @@ func (p Params) IsEVMChannel(channel string) bool {
 // IsActivePrecompile returns true if the given precompile address is
 // registered as an active precompile.
 func (p Params) IsActivePrecompile(address string) bool {
-	_, found := sort.Find(len(p.ActivePrecompiles), func(i int) int {
-		return strings.Compare(address, p.ActivePrecompiles[i])
+	_, found := sort.Find(len(p.ActiveStaticPrecompiles), func(i int) int {
```

Comment:
This is wrong because the sort.Find assume that the slice is sorted, which is not guarantee in our case. Look https://goplay.tools/snippet/8LkBItpBYo3

The function is also not used right?